OWNERS.md

```
@@ -0,0 +1,14 @@
+# Owners
+
+## Approvers
+
+- Shawn Hurley (https://github.com/shawn-hurley)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)